Web/Pages/CCGMember/Compare/ObjectiveLevel.cshtml

```diff
@@ -0,0 +1,58 @@
+﻿@page
+@using Microsoft.AspNetCore.Http
+@model Web.Pages.CCGMember.Compare.ObjectiveLevelModel
+@inject IHttpContextAccessor contextAccessor
+@inject IViewLocalizer Localizer
+@{
+    var isEnglish = System.Threading.Thread.CurrentThread.CurrentCulture.TwoLetterISOLanguageName == "en";
+    var id = Model.Id;
+    var jobGroupPositions = Model.JobGroupPositions;
+    var JobGroupName = isEnglish ? Model.JobGroup.NameEng : Model.JobGroup.NameFre;
+    var levels = jobGroupPositions.Select(e => e.LevelValue).ToList().Distinct();
+
+}
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
```

Comment:
Hi Nick, I am trying to get the dropdowns working without the bootstrap references using WET. Do you perhaps have an example of a WET dropdown that has the asp-page and route-data functionality?

Reply:
I don't have a specific example to share, but I did check the page and the dropdown toggle doesn't work without the bootstrap/jquery lines. Is there a reason you are using the dropdown toggle instead of a normal select tag?